wallet.js

```diff
@@ -1086,6 +1093,18 @@ function sendMultiPayment(opts, handleResult)
 	});
 }
 
+function readDeviceAddressesUsedInSigningPaths(onDone){
+	db.query(
+		"SELECT distinct device_address FROM shared_address_signing_paths UNION SELECT distinct device_address FROM wallet_signing_paths", 
```

Comment:
please use uppercase for sql keywords (DISTINCT)
yes add pending_shared_... even though they are not used now